src/azure-cli/azure/cli/command_modules/appservice/tests/latest/sample_web/package.json

```diff
@@ -2,6 +2,7 @@
   "name": "sample-node-web",
   "version": "1.0.0",
   "dependencies": {
-    "express": "^4.14.0"
+    "express": "^4.14.0",
+    "sample-node-web": "file:"
```

Comment:
May I ask why we need to modify package-lock.json and package.json files?

Reply:
these aren't manual changes, I just live run some tests and they're modified.

Reply:
It is better to exclude these changes from git. 
Please revert the changes of these two files and rerun relative tests in recording mode.